presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -281,6 +282,22 @@ public boolean supportsType(Type type)
     private Set<Format> formats = ImmutableSet.of();
     private Set<CompressionKind> compressions = ImmutableSet.of();
     private boolean useSelectiveOrcReader;
+    private boolean flattenAllColumns;
+
+    public static OrcTester quickDwrfFlatMapTester()
+    {
+        OrcTester orcTester = new OrcTester();
+        orcTester.nullTestsEnabled = true;
+        orcTester.skipBatchTestsEnabled = true;
+        orcTester.formats = ImmutableSet.of(DWRF);
+        orcTester.compressions = ImmutableSet.of(ZLIB);
+        orcTester.dwrfEncryptionEnabled = true;
+        orcTester.flattenAllColumns = true;
+
+        // TODO: sdruzkin - enable this flag after fixing a bug with null/empty maps in the selective reader
+        // orcTester.useSelectiveOrcReader = true;
```

Comment:
SelectiveReader doesn't handle columns with only null/empty maps present, reader treats such columns as `all nulls`. I'll post a PR to fix it.